client/src/App.js

```diff
@@ -7,15 +7,58 @@ import Footer from "./components/Footer";
 import Confirm from "./components/Confirm";
 import Calendar from "./components/Calendar";
 import Reports from "./components/Reports";
+import Add from "./components/Add";
+import { addHelper } from "./utils/addHelper.js";
+
 import { BrowserRouter, Route } from "react-router-dom";
 import data from "./data.json";
 import "typeface-roboto";
 
 class App extends Component {
   state = {
-    carData: null
+    carData: 1,
```

Comment:
`carData` seems like a bad name for this `carId` instead?

Reply:
Done